entity-framework/core/miscellaneous/testing/sqlite.md

```diff
@@ -43,4 +43,8 @@ The simplest way to enable testing against a different database is to modify you
 
 The key to testing with this provider is the ability to tell the context to use SQLite, and control the scope of the in-memory database. The scope of the database is controlled by opening and closing the connection. The database is scoped to the duration that the connection is open. Typically you want a clean database for each test method.
 
+>[!TIP]
+> To use `SqliteConnection()` reference the NuGet package `Microsoft.Data.Sqlite`.
```

Comment:
I would make it simply 

> To use `SqliteConnection()` and the `.UseSqlite()` extension method, reference the NuGet package `Microsoft.EntityFrameworkCore.Sqlite`.

Although SqlIteConnection is part of the ADO.NET provider, in practice you only need to add the EF Core provider package to make both work.

Reply:
Thanks for the explanations! I was confused because in my example I added first the Nuget package for Microsoft.Data.Sqlite and later Microsoft.EntityFrameworkCore.Sqlite soI was wrong!

Here we have the [new pull request ](https://github.com/aspnet/EntityFramework.Docs/pull/1427)